src/mithril.postgrest.js

```diff
@@ -1,12 +1,12 @@
 (function (factory) {
     if (typeof exports === 'object') {
         // Node/CommonJS
-        factory(require('mithril'), require('underscore'), require('node-localstorage'));
+        factory(require('mithril'), require('node-localstorage'));
     } else {
         // Browser globals
-        factory(m, _, localStorage);
```

Comment:
Hehe. true. Need to check how we are handling dependencies on those tests. 
This was actually an experimental branch. Shouldn't have pushed those changes. Was testing the whole integration: catarse_admin, mithril.postgrest, catarse. Will close this PR and send one just changing the token function.